zaza/openstack/charm_tests/glance/setup.py

```diff
@@ -87,11 +93,20 @@ def add_lts_image(glance_client=None, image_name=None, release=None):
     :param release: Name of ubuntu release.
     :type release: str
     """
+    image_arch = os.environ.get('image_arch')
```

Comment:
Tox will only allow env variable with a certain perfix to be passed through so image_arch won't be present even if you set it before running the test. People have the choice of setting deployment context in a config file or in environment variables. To maintain this support please use `get_deployment_context` rather than accessing os.environ directly.

Something like:

```
import zaza.utilities.deployment_env as deployment_env


deploy_ctxt = deployment_env.get_deployment_context()
image_arch = deploy_ctxt.get('TEST_IMAGE_ARCH', 'amd64'))
```